spec/lib/data_update_scripts/resync_elasticsearch_documents_spec.rb

```diff
@@ -1,7 +1,12 @@
 require "rails_helper"
 require Rails.root.join("lib/data_update_scripts/20200410152018_resync_elasticsearch_documents.rb")
 
-describe DataUpdateScripts::ResyncElasticsearchDocuments, elasticsearch: true do
+describe DataUpdateScripts::ResyncElasticsearchDocuments, elasticsearch: %w[FeedContent User Tag] do
+  after do
+    Article::SEARCH_CLASS.refresh_index
```

Comment:
We only need to refresh after specs that have bulk updates so rather than do this for every Elasticsearch spec I choose to keep it to the ones that were bulk update specific